From dea9cf357741592785a378504b465b4413884443 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Thu, 22 Feb 2024 18:01:08 +0100
Subject: [PATCH 0644/1695] dts: stm32: enable hse monitoring for
 stm32mp135f-dk boards

This commit activates hse monitoring on stm32mp135f-dk boards.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 625a2efe7f808841728933eb6cc6ed62d591ee71 dts: stm32: enable hse monitoring for stm32mp135f-dk boards]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: I08e0f2383c1aeaf5e7e5a6ecaffac1373743c46c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/362462
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index 4f5917ce5..71e918162 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -163,6 +163,10 @@
 	st,protreg = < (TZPROT(15)) >;
 };
 
+&hse_monitor {
+	status = "okay";
+};
+
 &i2c4 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c4_pins_a>;
-- 
2.47.2

